detox/android/detox/src/main/java/com/wix/detox/espresso/action/DetoxMultiTap.kt

```diff
@@ -37,8 +37,9 @@ open class DetoxMultiTap
             private val longTapMinTimeMs: Long = getLongTapMinTime(),
             private val tapEvents: TapEvents = TapEvents(),
             private val uiControllerCallSpy: UiControllerSpy = UiControllerSpy.instance,
-            private val log: DetoxLog = DetoxLog.instance)
-    : Tapper {
+            private val log: DetoxLog = DetoxLog.instance,
+            private val duration: Long? = null
```

Comment:
I'm thinking to rename `DetoxMultiTap` to `DetoxCustomTapper` / `DetoxTapper` since it will better reflect its role.

Reply:
Done.